archivebox/index/html.py

```diff
@@ -31,6 +31,7 @@
 
 join = lambda *paths: os.path.join(*paths)
 MAIN_INDEX_TEMPLATE = join(TEMPLATES_DIR, 'main_index.html')
+MINIMAL_INDEX_TEMPLATE = join(TEMPLATES_DIR, 'main_index_minimal.html')
```

Comment:
Why can't the minimal template just be a subset of the main template?

e.g.

- `main_index.html`
   ```html
   <html>
       <head>
           ...
       </head>
       <body>
           <header>...</header>
           <main>
               {% include 'main_index_table.html' %}
           </main>
           <footer>...</footer>
       </body>
- `main_index_table.html`

Reply:
These templates don't use django's rendering system. We should change them to use it, but that could be done in a later version.